util/src/main/java/tech/pegasys/artemis/util/mikuli/PublicKey.java

```diff
@@ -69,22 +73,42 @@ public static PublicKey fromBytesCompressed(byte[] bytes) {
    * @return a valid public key
    */
   public static PublicKey fromBytesCompressed(Bytes bytes) {
-    G1Point point = G1Point.fromBytesCompressed(bytes);
-    return new PublicKey(point);
+    checkArgument(bytes.size() == 48, "Expected 48 bytes of input but got %s", bytes.size());
```

Comment:
If we're potentially getting completely invalid data, will this check cause us trouble?

Reply:
Yeah, it's redundant in any case, so removing.